boa/src/exec/mod.rs

```diff
@@ -140,7 +142,10 @@ impl Interpreter {
             ValueData::Rational(rational) => Ok(Number::to_native_string(*rational)),
             ValueData::Integer(integer) => Ok(integer.to_string()),
             ValueData::String(string) => Ok(string.clone()),
-            ValueData::Symbol(_) => panic!("TypeError exception."),
+            ValueData::Symbol(_) => {
+                self.throw_type_error("TypeError: \"can't convert symbol to string\"")?;
```

Comment:
Should we just add the `TypeError` part to all the `TypeError` messages when printing?

Reply:
Yes. you are right. The message will be `TypeError: "TypeError: \"can't convert symbol to string\""`, I'll change it :)